src/draw/lv_draw_buf.h

```diff
@@ -48,7 +48,7 @@ LV_EXPORT_CONST_INT(LV_STRIDE_AUTO);
  * For platform that needs special buffer alignment, call LV_DRAW_BUF_INIT_STATIC.
  */
 #define LV_DRAW_BUF_DEFINE_STATIC(name, _w, _h, _cf) \
-    static uint8_t buf_##name[LV_DRAW_BUF_SIZE(_w, _h, _cf)]; \
+    static LV_ATTRIBUTE_MEM_ALIGN uint8_t buf_##name[LV_DRAW_BUF_SIZE(_w, _h, _cf)]; \
```

Comment:
How about we phase out the macro, and move to dynamic alloc.

Reply:
IMO it would lead to hard to find our of heap issues. I like the explicitness of static allocations and that we can better see the RAM usage at compile time. 

So I think  adding `LV_ATTRIBUTE_MEM_ALIGN` here is a good idea. We have `LV_ATTRIBUTE_MEM_ALIGN` exactly fro these use cases.